File: min_num.go
package main
import "fmt"

func main() {
	var min int
	fmt.Scan(&min)
	for i := 1; i < 5; i++ {
		var num int
		fmt.Scan(&num)
		if num < min {
			min = num
		}
	}
	fmt.Println(min)
}

File: min_num.cs
using System;
using System.Linq;

public class Program
{
	public static void Main()
	{
		int[] num = new int[5];
		for (int i = 0; i < num.Length; i++) num[i] = int.Parse(Console.ReadLine());
		Console.WriteLine(num.Min());
	}
}
